modules/restore-determine-state-etcd-member.adoc

```diff
@@ -70,7 +70,7 @@ $ oc get nodes -l node-role.kubernetes.io/master | grep "NotReady"
 .Example output
 [source,terminal]
 ----
-ip-10-0-131-183.ec2.internal   NotReady   master   122m   v1.22.0 <1>
+ip-10-0-131-183.ec2.internal   NotReady   master   122m   v1.22.1 <1>
 ----
 <1> If the node is listed as `NotReady`, then the *node is not ready*.
```

Comment:
The information payload on this callout seems low.

Reply:
Possibly, but that's out of scope of this PR.